leftwm-core/src/models/screen.rs

```diff
@@ -8,7 +8,7 @@ use x11_dl::xlib;
 pub struct Screen {
     pub root: WindowHandle,
     pub output: String,
-    pub id: Option<WorkspaceId>,
+    pub id: WorkspaceId,
```

Comment:
What's the reasoning? See comments above. I think an absent value might be more straightforward than a special handling when it is `0`.

Reply:
I just don't like Options where they are not absolutely necessary, because of the extra handling needed, and in this case there will always be an id assigned directly after object creation. That's all, I don't have a strong opinion on this and can change it back if you prefer it otherwise.

Reply:
I see, yeah I'd appreciate it if you could change it back to Option. I prefer it being `None` rather than an "invalid" value `0`